File: 8kyu/max-and-min-of-arr/solution.cs
public class Kata
{
		public int Min(int[] list) {
        int min = list[0];
        for (int i=0;i<list.Length;i++)
        {
            if (list[i] < min) min = list[i];
        }
        return min;
		}
  
		public int Max(int[] list){
				int max = list[0];
        for (int j=0;j<list.Length;j++)
        {
            if (list[j] > max) max = list[j];
        }
        return max;
		} 
}

File: 8kyu/max-and-min-of-arr/solution.java
public class Kata {

  public int min(int[] list) {
    int minimum = list[0];
    for (int i = 0;i<list.length;i++){
      if(list[i] < minimum) minimum = list[i];
    }
    return minimum;
  }

  public int max(int[] list) {
    int maximum = list[0];
    for (int i=0;i<list.length;i++){
      if (list[i] > maximum) maximum = list[i]; 
    }
    return maximum;
  }
}


//top rated
import java.util.Arrays;

public class Kata {

  public int min(int[] list) {
    return Arrays.stream(list).min().getAsInt();
  }
  
  public int max(int[] list) {
    return Arrays.stream(list).max().getAsInt();
  }
}
